src/applications/disability-benefits/2346/components/OrderSupplyPageContent.jsx

```diff
@@ -1,6 +1,6 @@
-import React from 'react';
+import React, { Component } from 'react';
 
-class orderSupplyPageContent extends React.Component {
+class OrderSupplyPageContent extends Component {
```

Comment:
We could change this to a functional component if we want since there's no state being used 🙂

Reply:
yassss